### DISCUSSION TITLE
Error: PowerGrid requires flatpickr for Date filters to work.

### DISCUSSION THREAD

**Discussion by bounty12** [Power-Components/livewire-powergrid#1433] [Q&A]
steps to reproduce

laravel new new-project
composer require laravel/jetstream
php artisan jetstream:install livewire --dark
composer require power-components/livewire-powergrid
Import powergrid JS and CSS
in resources/js/app.js
import './../../vendor/power-components/livewire-powergrid/dist/powergrid'
 
// If you use Tailwind 
import './../../vendor/power-components/livewire-powergrid/dist/tailwind.css'

php artisan powergrid:create

 
🤔 It looks like your project is missing a dependency!

PowerGrid requires flatpickr for Date filters to work.

Please, consider installing the package.

For more information, visit: https://livewire-powergrid.com/table/column-filters.html#filter-datetimepicker

**Comment by dansysanalyst** [comment]
it is a suggestion to install flatpickr

**Reply by bounty12** [reply]
It was never required to install separately, I am working on project created many powergrid table, suddenly this error starts showing up, then I tried on completely new project  to create a powergrid component even then i am getting this error, and yes I have also installed flatpickr as told in this instruction "https://livewire-powergrid.com/table/column-filters.html#filter-datetimepicker" even then the error is same
